java/maven/nbproject/project.xml

```diff
@@ -636,6 +592,7 @@
                 <package>org.netbeans.modules.maven.api.execute</package>
                 <package>org.netbeans.modules.maven.api.output</package>
                 <package>org.netbeans.modules.maven.api.problem</package>
+                <package>org.netbeans.modules.maven.configurations</package>
                 <package>org.netbeans.modules.maven.execute</package>
```

Comment:
This is not good idea. While publishing `org.netbeans.modules.maven.api.*` is OK, the `org.netbeans.modules.maven` shall probably stay as friend.

Reply:
AFAIK both friend and public APIs can't be defined in the same module.

Reply:
That is true. The module would have to be split into two. The API and the implementation.

Reply:
Try to refactor the functionality you need into `maven.api` module. A bit more work, but it will pay off for the whole Apache NetBeans project.